src/ledger/insert_shred.zig

```diff
@@ -829,6 +898,9 @@ pub const ShredInserter = struct {
         const shred_index_u32 = shred.fields.common.index;
         const shred_index_u64: u64 = @intCast(shred_index_u32);
         const is_last_in_slot = shred.isLastInSlot();
+        // QUESTION/SUGGESTION rename this to is_valid (or similar) instead of sanitize as santize gives the
```

Comment:
sanitize is what is used throughout agave so originally we started to copy their approach which has led to many sanitize method calls everywhere (mainly gossip since it was the first module we worked on) -- imo i agree we should stop using sanitize, ideally change it to `validate` (see `AccountInFile.validate`)